sublime-package.json

```diff
@@ -6,13 +6,13 @@
           "/LSP-rust-analyzer.sublime-settings"
         ],
         "schema": {
-          "$id": "sublime://settings/LSP-rust-analyzer",
+          "$id": "sublime://settings/rust-analyzer",
```

Comment:
I'd keep the schema ID to closely match the package name though. No chances of conflicts between packages if we follow that.